### ISSUE TITLE
[Bug]: WebRTC Acceleration/Deceleration Changes Pitch

### ISSUE THREAD

**Issue by Jamie0** [electron/electron#41158]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

29.0.0-beta3

### What operating system are you using?

macOS

### Operating System Version

Tested on macOS 10.15 on x64, macOS 13 on Apple Silicon

### What arch are you using?

Other (specify below)

### Last Known Working Electron version

17.0.0

### Expected Behavior

When the WebRTC jitter buffer accelerates/decelerates audio (inside an AudioContext), due to network condition changes, the playback pitch should be consistent. 

### Actual Behavior

When using WebRTC inside Electron (within an AudioContext), the playback pitch is not consistent, making it unsuitable for broadcast/music monitoring. 

### Additional Information

This originally seemed like a [bug](https://bugs.chromium.org/p/chromium/issues/detail?id=1522080) in Chromium itself - but I can't reproduce it in the latest builds of Chrome or Chromium. I don't have a reproduction example to hand outside of our application, but I'll try to create one and add it to this issue. 

It seems the acceleration/deceleration feature in Blink's jitter buffer changed around M100 to adjust the rate, replacing the old behaviour of inserting silent/removing samples.

**Comment by codebytere** [comment]
@Jamie0 is there a way you could possibly attach a reproducible sample for us to test and better investigate with? That'd make it easier for us to dig into this :)

**Comment by Jamie0** [comment]
@codebytere

I've [uploaded](https://gist.github.com/Jamie0/84e22d2164843a01564a6ee405be7ed8) a bare minimum test case - two connections in one tab, playing out via an AudioContext, with some simulated jitter.

But as of this morning, I'm not observing any pitch bend on my MacBook or the 'lab' Mac Mini, which is bizarre, as when I tested this on Monday, Electron on both machines was reliably producing audio with pitch shifting every few seconds, where Chrome was producing a stable stream. 

I'm going to close this issue for now, until I can be 100% sure I've ruled out an environmental cause (i.e. an issue on the sender).

Nonetheless, I hope the test case is useful, as I don't think there are any test cases that combine PeerConnection and AudioContext.
